### PR TITLE
Make implicit proposer vote explicit

### LINKED ISSUE #250: When creating proposals, the proposer's implicit yes vote is unhelpful
When a new governance proposal is created through the member frontend, the proposing member is assumed to vote for it, unless they submit an explicit vote separately. This is not obvious. All members must know about this behaviour as the vote is not stored or visible in queries, and any code changes around these votes must work with this implicit behaviour. There are also use cases where the proposer's vote is not an automatic yes, it may be as conditional on service state as any other vote, but the proposal could be accepted before the proposer can vote.

Instead, we should allow the proposer to submit their voting ballot _with_ the proposal creation, and the default value for this ballot if unspecified is `return true`. This ballot is stored in the votes table so can be retrieved and processed like any other ballot, simplifying the counting logic and making the proposer's vote visible to all other members.

### PR DESCRIPTION
**Pull request by eddyashton**
The core of this is just adding a ballot to `Proposal::In`, recording it, and simplifying the logic of `complete_proposal`. I also added a unit test of the new behaviour (its possible to vote no with your initial proposal). While fixing references in the documentation I noticed that `governance.rst` was out of date, so I've corrected the command line examples to their current format.

Resolves #250.

### CODE REVIEW

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

File: sphinx/source/governance.rst [hunk 1]
```diff
@@ -44,34 +44,50 @@ As soon as ``member3`` accepts the proposal, a quorum (2 out of 3) of members ha
...
-    ------ Proposal ------
-    -- Proposal id: 1
-    -- Proposer id: 0
-    -- Script: {"text":"\n      tables, member_cert = ...\n      return Calls:call(\"new_member\", member_cert)\n    "}
-    -- Parameter: [<member_cert>]
-    -- Votes: [[1,{"text":"\n      tables, changes = ...\n      return false"}]]
-    ----------------------
-
-In this case, there is one pending proposal (``id`` is 1), proposed by the first member (``member1``, ``id`` is 0) and which will call the ``new_member`` function with the new member's certificate as a parameter. Only one vote has been cast by ``member2`` (``id`` is 1) to reject the proposal while ``member1`` (proposer) has already implicitly accepted it.
+    $ memberclient --server-address 127.83.203.69:55526 --cert=member1_cert.pem --privk=member1_privk.pem --ca=networkcert.pem proposal_display
```
Comment (jumaffre) [inline sphinx/source/governance.rst]: Should we stick to `--option=` (instead of `--option `) for this doc?

File: sphinx/source/governance.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ For example, ``member1`` may submit a proposal to add a new member (``member4``)
 
 .. code-block:: bash
 
-    $ memberclient add_member --ca=networkcert.pem --member_cert=member4_cert.pem --cert=member1_cert.pem --privk=member1_privk.pem --host=10.1.0.4 --port=25000
+    $ memberclient --server-address 127.83.203.69:55526 --cert member1_cert.pem --privk member1_privk.pem --ca networkcert.pem add_member --member_cert member4_cert.pem
```
Comment (jumaffre) [inline sphinx/source/governance.rst]: `--member_cert` -> `--member-cert` (dash)

File: sphinx/source/governance.rst (hunk 1 above)
Comment (eddyashton) [reply sphinx/source/governance.rst]: Have opted for consistency on `--option value`.

**Comment by cimetrics** [comment]
images

**Pull request merged**